systemd/ceph-rgw.tmpfiles.d

```diff
@@ -0,0 +1,2 @@
+# create rgw socket directory
+d /var/run/ceph-rgw 0755 wwwrun www - -
```

Comment:
I can template the user name in autotools.

SUSE has run out of fixed UID's we have to do these more dynamically and use user names.
I think the path to the radosgw socket is in the ceph.conf file.

I think its relatively trivial to run ceph as non root. Its moderately high up my list of requirements after getting the spec file done, what I am more unsure about is how to get the cgroup for ceph really locked down so ceph can only be ceph even if compromised.